requirements-dev.txt

```
@@ -1 +1,2 @@
-dash[ci,dev,testing]==2.6.2
\ No newline at end of file
+dash[ci,dev,testing]==2.6.2
+pytest<7.2.0
```

```suggestion
dash[ci,dev,testing]==2.7.0
```
https://github.com/plotly/dash/pull/2287